airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -19,6 +19,15 @@
 Changelog
 ---------
 
+2.0.0
```

Comment:
Could you please remove the changelog entry :) . I am fully automating changelog preparation for the next release and from then on there will be no need to add any links here by committers, except if there will be a breaking change requiring explanation.

Reply:
@potiuk updated